source/robot_model/include/robot_model/Model.hpp

```diff
@@ -71,6 +71,13 @@ class Model {
    */
   state_representation::CartesianPose forward_geometry(const state_representation::JointState& joint_state,
                                                        unsigned int frame_id);
+  
+  /**
+   * @brief It checks if the vector's elements are inside at their limits
+   * @param vector the vector we want to analyze
```

Comment:
> the vector we want to analyze

I think in general we use "third person" language in documentation, so not using I / we / you. The fix to this would be 

"the vector to be analyzed / the vector to check"